image-builder/image_builder.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+if [ -n "$DEBUG" ] ; then
+	set -x
+fi
+
+SCRIPT_NAME="${0##*/}"
+IMAGE="kata-containers.img"
+AGENT_BIN=${AGENT_BIN:-kata-agent}
+MOUNT_DIR="/tmp/kata-mount-image"
+
+die()
+{
+	msg="$*"
```

Comment:
Nit/idea: you could use `local` for all the variables defined in functions to avoid potential surprises.